autogen/docs/programming.md.mustache

```diff
@@ -368,14 +367,41 @@ Since many colors are missing from the NetLogo color space, [[approximate-hsb]]
 and [[approximate-rgb]] often can't give you the exact color you ask for, but
 they try to come as close as possible.
 
-Example: you can change any turtle from it's existing NetLogo color to a half
-transparent version of that color using:
+Example:
 
-    set color lput 125 extract-rgb color
+    let my-color approximate-rgb 0 0 255 ;; my-color is now 104.7
+    show extract-rgb my-color ;; shows [48 88 161] which is pretty far from [0 0 255], the color we started with
+
+This is an approximation, but it is still near NetLogo blue which is 105.
+
+Using RGBA colors, the user has more options when it comes to an agent's color.
+
+Example: you can change any turtle from its existing NetLogo color to a sixty percent transparent version of that color using:
+
+    ask one-of turtles [set color lput 102 extract-rgb color]
+
+Note: because 255 is fully opaque and sixty percent transparent is equal to forty percent opaque, the correct alpha value is  255 * .4 = 102.
+See the palette extension section below for an alternate implementation.
 
 > **Code Examples:** HSB and RGB Example (lets you experiment with the HSB and
 > RGB color systems), Transparency Example
 
+**Palette Extension**
+
+The palette extension offers primitives that give the user more control over colors. It allows for direct
+manipulation of RGBA and HSB components without the requiring list manipulations.
+
+Example: changing the transparency of any turtle to sixty percent transparent with the palette extension:
+
+    extensions [palette]
+    ask turtle turtle-number [palette:set-transparency 60]
+
+The extension offers similar commands for Red, Green, Blue, Alpha, Hue, Saturation, and Brightness.
+
+See the [documentation](https://ccl.northwestern.edu/netlogo/docs/palette.html) of the palette extension for more information.
```

Comment:
Thanks for adding the link, but it should be relativized for use by the offline docs as well.  I see I forgot to paste the example into my original comment, sorry about that!  https://github.com/NetLogo/NetLogo/blob/hexy/autogen/docs/programming.md.mustache#L1973